### PR TITLE
EE44 - Moving Development Content

### PR DESCRIPTION
**Pull request by tst-ahernandez**
https://github.com/google/earthenterprise/issues/44
Moved the development content (build instructions) off the opengee page and provided a link to the main GitHub page in its place. All the content is already in the Github wiki (and up to date) so any user can quickly navigate and find it there.

Also, added a small fix so that when the Develop tab is selected on opengee the "Develop" tab is highlighted. This previously always highlighted Documentation.

To verify this change
1. Go to: https://tst-ahernandez.github.io/earthenterprise/index.html
2. Navigate to the Develop tab
3. Verify that the page content is updated with all the build instructions removed

### CODE REVIEW

**Comment by tst-ccamp** [comment]
Hey @tst-ahernandez I think the intent was to also move the other content (GETTING STARTED WITH GOOGLE EARTH ENTERPRISE and HOW TO CONTRIBUTE) to the Wiki, since they are developer docs and only use opengee.org for user docs.

File: development/getting-started.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 layout: page
 title:  "Getting Started With Google Earth Enterprise"
 icon: globe
+section_id: develop
 ---
 
 Earth Enterprise is the open source release of Google Earth Enterprise, a geospatial application which provides the ability to build and host custom 3D globes and 2D maps. Earth Enterprise does not provide a private version of Google imagery that's currently available in Google Maps or Earth.
```
Comment (tst-ccamp) [inline development/getting-started.md]: Can you also move the rest of the sections on this page to the Wiki?

**Comment by tst-ccamp** [comment]
Can you also add a link to the description of the PR to the issue this is fixing? That will automatically tie the two together under the issue history.

File: development/getting-started.md (hunk 1 above)
Comment (tst-ahernandez) [inline development/getting-started.md]: Moved to the wiki!

**Comment by tst-ahernandez** [comment]
Updated per the previous comments. Ready for testing.

File: development/getting-started.md [hunk 2]
```diff
@@ -16,136 +17,7 @@ The application suite consists of three core components:
 
 `Client` - The Google Earth Enterprise Client (EC) and Google Maps Javascript API V3 used to view 3D globes and 2D maps, respectively.
```
Comment (andreisel) [inline development/getting-started.md]: Maybe re-phrase:
The application suite consists of four core components:

`Fusion` - Builds 3D Earth and 2D Map from imagery, vector and terrain source data. 
 
 `Earth Enterprise Server` - Apache -based server which hosts 3D Earth, 2D Map build by Fusion and Portable 3D/2D globes.
 
 `Portable server` - Tornado-based server which hosts  Portable 3D/2D globes in a local-server configuration.

`Client` - The Google Earth Enterprise Client (EC) and Google Maps Javascript API V3 used to view 3D globes and 2D maps, respectively.

**Comment by tst-ccamp** [comment]
@tst-ahernandez I'll start by apologizing about being nit-picky here. :) This isn't really a ding against the PR, but may effect the link. But, I think on the Wiki, either the content on the Getting started page and the Home page should be swapped (Getting started section is more like an intro) or it should all be combined on the Home page. And if you decide to move "How to Contribute" to its own page, it should probably just be called "How to Contribute" rather than anything else. I think the simplest thing to do would be to move everything to Home and link there, and get rid of "Getting Started".

Also, can you update the wording according to Andrei's suggestion?

**Comment by tst-ahernandez** [comment]
I've updated the wiki with the changes as requested.

**Comment by tst-ccamp** [comment]
The only thing left to do is to update the link on the website to point to the new wiki page

**Comment by tst-ahernandez** [comment]
Updated!

**Pull request merged**